test/libgit2.jl

```diff
@@ -246,6 +246,8 @@ mktempdir() do dir
                 else
                     error("Found unexpected entry: $name")
                 end
+                show_str = sprint(show, entry)
+                @test show_str == string("ConfigEntry(\"", name, "\", \"", value, "\")")
```

Comment:
Just a style issue but I find using string interpolation makes things easier to read: 
```julia
"ConfigEntry(\"$name\", \"$value\")"
```